Mathlib/CategoryTheory/Abelian/Pseudoelements.lean

```diff
@@ -460,7 +460,7 @@ variable [Limits.HasPullbacks C]
     that `f p = g q`, then there is some `s : pullback f g` such that `fst s = p` and `snd s = q`.
 
     Remark: Borceux claims that `s` is unique, but this is false. See
-    `Counterexamples/Pseudoelement` for details. -/
+    `Counterexamples/Pseudoelement.lean` for details. -/
```

Comment:
There is no way to make this clickable in the doc, right?

Reply:
This works now: https://github.com/leanprover/doc-gen4/pull/151